yarn-project/prover-client/src/orchestrator/orchestrator_public_functions.test.ts

```diff
@@ -56,93 +54,5 @@ describe('prover/orchestrator/public-functions', () => {
         expect(block.number).toEqual(context.blockNumber);
       },
     );
-
-    it('nested public calls', async () => {
```

Comment:
test doesn't make sense any more because we aren't generating proofs per nested public call